docs/common/istio-ambient-waypoint.adoc

```diff
@@ -128,12 +132,169 @@ spec:
+
+1. After the new Istio control plane revision is ready, verify waypoint proxy pods are running:
++
+[source,bash,subs="attributes+",name="ambient-waypoint-update-revisionbased"]
+----
+$ kubectl get pods -n bookinfo -l gateway.networking.k8s.io/gateway-name=waypoint
+NAME                       READY   STATUS    RESTARTS   AGE
+waypoint-5d9c8b7f9-abc12   1/1     Running   0          5m
+----
+
+2. Confirm the waypoint proxy is connected to the new control plane revision:
++
+[source,bash,subs="attributes+",name="ambient-waypoint-update-revisionbased"]
+----
+$ istioctl proxy-status | grep waypoint
+waypoint-5d9c8b7f9-abc12.bookinfo     SYNCED     SYNCED     SYNCED     SYNCED     istiod-1-26-4-7b9f8c5d6-xyz78.istio-system     1.26.4
+----
++
+This command queries the Istio control plane to verify that the waypoint proxy is connected to the new revision. The output shows the waypoint proxy is now connected to the revision-specific istiod pod (with revision `1-26-4` in its name) and is running the updated version (1.26.4). The revision-specific naming in the ISTIOD column confirms the waypoint has successfully migrated to the new control plane revision.
+
+3. _(Optional)_ Update waypoint proxy pods to use the new proxy version:
++
+Waypoint proxy pods automatically use the correct proxy version when they reconnect to the new control plane revision. In most cases, the waypoint pods will update automatically during the control plane migration. However, you can manually trigger an update using one of the following approaches:
++
+**Option A: Restart waypoint pods (Recommended)**
++
+Restart the waypoint deployment to trigger the pods to reconnect to the new control plane and pick up the updated proxy version:
++
+[source,bash,subs="attributes+",name="ambient-waypoint-update-revisionbased"]
+----
+$ kubectl rollout restart deployment -n bookinfo -l gateway.networking.k8s.io/gateway-name=waypoint
+----
++
+This approach allows the waypoint to automatically use the appropriate proxy version from the new control plane revision.
++
+**Option B: Explicitly set proxy image version**
++
+If you need to pin the waypoint to a specific proxy version (for example, to use a version different from the control plane or for testing purposes), you can explicitly patch the Gateway resource:
++
+[source,bash,subs="attributes+",name="ambient-waypoint-update-revisionbased"]
+----
+$ kubectl patch gateway waypoint -n bookinfo --type='merge' -p '{"spec":{"template":{"spec":{"containers":[{"name":"istio-proxy","image":"docker.io/istio/proxyv2:1.26.4"}]}}}}'
+----
++
+NOTE: Manually setting the proxy image version is only necessary when you want to pin to a specific version different from what the control plane would automatically provide, or in scenarios where automatic updates do not occur due to environmental or configuration constraints. In standard RevisionBased updates, Option A (restart) is sufficient.
```

Comment:
afaik neither restart nor manual version updates should be necessary. istiod will reconcile the deployment after it has been updated, and update the waypoint